qiskit/providers/ibmq/api/ibmqconnector.py

```diff
@@ -50,7 +50,7 @@ def get_backend_defaults_url(config, backend_type):
     return '/Backends/{}/defaults'.format(backend_type)
 
 
-def get_backends_url(config, hub, group, project):
+def get_backends_url(config, hub=None, group=None, project=None):
```

Comment:
Actually, seeing that the only place where this function is called (at `IBMQConnector.available_backends()`) is done with only the `config` parameter, can you remove `hub`, `group`, and `project` from this function as well, adjusting the checks inside accordingly, as in:

```python
def get_backends_url(config):
    """Return the URL for a backend."""
    hub = config.get('hub', None)
    group = config.get('group', None)
    project = config.get('project', None)

    if hub and group and project:
    ...
```

And while we are at it, can you do the same for `get_backend_properties_url()` and `get_job_url()`? This would make all the four "URL" methods consistently depend only on `config` for authentication-related values.

Reply:
Thanks for all the tips @diego-plan9  ... will take into account in future.
Committed changes and pushed.

Reply:
Nice, thanks!